### ISSUE TITLE
The Prime Minister's Speech(es)

### ISSUE THREAD

**Issue by gabri985** [Milano-R/erum2020-covidr-contest#86]
Submit your contributions by filling the following information

``` yaml
# Contribution's title, author, abstract
title: The Prime Minster's Speech(es)
author: Gabriele Galatolo, Ilaria Ceppa, Francesca Giorgolo, Matteo Papi, Andrea Zedda, Marco Calderisi
abstract: |
  Italy was the first European country to face the Covid19 crisis and is one of the most hit by the pandemic. This led the Italian Government to take extraordinary decisions and measures to manage the economic and social effects related to the contagions.
  In this article we posted on our "Kode Covid19 Blog" we analyzed the speeches given by the Italian Prime Minister Giuseppe Conte comparing them with the reactions we retrieved from Italian Twitter messages having crisis-related hashtags (e.g. #coronavirus, #covid19).
  With simple text-mining techniques and sentiment analysis we provided visualizations and analysis about the crisis and the population feelings seen from the point of view of both Twitter and the Prime Minister conferences.

# Short name to be used in the website Gallery menu (can be the same as title)
menu_entry: The Prime Minster's Speech(es)
# URL of the public source code repository
repository: https://gitlab.com/kode_public/blogdown_website/-/tree/master/content/articles/2020-03-24-speeches-of-the-prime-minister
type: gist
content: <script src="https://gist.github.com/gabri985/214a998e7d270e58303519383b3ebf7a.js"></script>
```

**Comment by riccardoporreca** [comment]
@gabri985, the repository https://gitlab.com/kode_public/blogdown_website/-/tree/master/content/articles/2020-03-24-speeches-of-the-prime-minister is not accesible, although https://gitlab.com/kode_public/ is.

In addition, we have to find a workaround the "mixed content" issue related to your `content` not being reachable via HTTPS. This unfortunately prevents the app from being embedded in the [CovidR Gallery](https://milano-r.github.io/erum2020-covidr-contest/), but should not be a hard blocker. Of course the ideal solution would be have HTTPS `content`

My simplest suggestion is to embed a screenshot with an external link to the actual content. The easy way to do so is to create a GitHub [Gist](https://gist.github.com/). Something along the lines of the example at https://gist.github.com/baranovskypd/97c671656ae2392991ed59884222aeca.
You can add any text in the Gist if yow want to supply more information do be displayed in the gallery page.

Sorry for the inconvenience, let us know if you have questions or would like to propose alternatives or discuss more in detail.

**Comment by gabri985** [comment]
Regarding the first point, we set the project as public and you should access it now. 

About the second issue, we are working to add HTTPS support: alternatively, we will follow your suggestion and we will create a GitHub Gist.

**Comment by gabri985** [comment]
@riccardoporreca we have some problems to add HTTPS support currently, so I updated the original issue with the yaml for the submission with the GIST script.
Can you please confirm that now it's all ok? Thanks!

**Comment by riccardoporreca** [comment]
> Can you please confirm that now it's all ok? Thanks!

@gabri985, thanks, the current status is currently fine for us to accept. I will proceed with integrating your contribution soon using the Gist.

**Comment by riccardoporreca** [comment]
@gabri985, one more thing, I will shorten the `author` field as follows for a better look in the gallery
```yml
author: Gabriele Galatolo et al., Kode Solutions
```

**Comment by gabri985** [comment]
Yes, is fine for us

**Comment by riccardoporreca** [comment]
There are currently issues with GitHub preventing the integration of contributions (https://www.githubstatus.com/).
We will finalize submissions once such issues are solved.

**Comment by riccardoporreca** [comment]
Merged corresponding PR #95.
Integration of contributions in the CovidR Gallery still blocked by an ongoing GitHub incident

**Comment by riccardoporreca** [comment]
@gabri985, the major GitHub incident was solved and now contributions are fully integrated.

Your contribution is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/kode-prime-minister-speech.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686
